spec.html

```diff
@@ -142,6 +153,11 @@ <h1>Conformance</h1>
   <p>A conforming implementation of ECMAScript may support program and regular expression syntax not described in this specification. In particular, a conforming implementation of ECMAScript may support program syntax that makes use of any &ldquo;future reserved words&rdquo; noted in subclause <emu-xref href="#sec-keywords-and-reserved-words"></emu-xref> of this specification.</p>
   <p>A conforming implementation of ECMAScript must not implement any extension that is listed as a Forbidden Extension in subclause <emu-xref href="#sec-forbidden-extensions"></emu-xref>.</p>
   <p>A conforming implementation of ECMAScript must not redefine any facilities that are not implementation-defined, implementation-approximated, or host-defined.</p>
+  <p>A conforming implementation of ECMAScript may choose to implement or not implement Normative Optional subclauses. However if Normative Optional behavior is implemented in some form, it must be implemented in accordance to the Normative Optional clause. A Normative Optional clause is defined in the spec within a Normative Optional colored box.</p>
```

Comment:
```suggestion
  <p>A conforming implementation of ECMAScript may choose to implement or not implement Normative Optional subclauses. If any Normative Optional behaviour is implemented, all of the behaviour in the containing Normative Optional clause must be implemented. A Normative Optional clause is denoted in the spec with the words "Normative Optional" in a coloured box, as shown below.</p>
```

Reply:
The spec doesn't refer to itself as "the spec". Usually it says "this specification", occasionally "this document".

Reply:
@jmdyck Updated.

Reply:
+1 to @michaelficarra's edit.